### DISCUSSION TITLE
Adding a number field with a default value to an existing entry doesn't use that default value

### DISCUSSION THREAD

**Discussion by jamiewade** [craftcms/cms#8600] [Ideas]
### Description

When you create a new number field with a 'default value' and add that field to an entry type, entries that were created **before** the field was created will not adhere to the default value, so the field will be blank.

The default value option does work for **new entries**, it just doesn't work for **existing entries**.

### Expected behaviour

The number field should be populated with whatever number is set as the 'default value', regardless of when the entry was created.

### Steps to reproduce

1. Create an entry
2. Create a new number field and set a value for the 'Default value' option. Add the field to the entry type used in step 1
3. Visit that entry with the field added, and you will see it is empty
4. Create a _new_ entry and you will see the field is populated with the default value

### Additional info

- Craft version: 3.0.27.1
- PHP version: 7.2.10
- Database driver & version: MySQL 5.7.19
- Plugins & versions: No plugins installed

**Comment by brandonkelly** [comment]
Your desired behavior isn’t possible right now, but it’s something we can consider making possible as part of #1524.
